### PR TITLE
feat: #6995 app market data fetching

### PR DESCRIPTION
**Pull request by willmcvay**
# Pull request checklist

**Detail as per issue below (required):**

fixes: #6995

### CODE REVIEW

File: packages/app-market/src/components/apps-browse/app-filters-collection.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import { AppsBrowseConfigItem } from './use-apps-browse-state'
+
+interface AppFiltersCollectionProps {
+  configItem: AppsBrowseConfigItem
+}
+
+export const AppFiltersCollection: FC<AppFiltersCollectionProps> = ({ configItem }) => {
+  const { filters, content } = configItem
+
+  console.log('App Filters: ', filters, content)
```
Comment (bashleigh) [inline packages/app-market/src/components/apps-browse/app-filters-collection.tsx:11]: do you need this?

File: packages/app-market/src/components/apps-browse/developer-apps-collection.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  const developerId = connectSession?.loginIdentity.developerId
+
+  const [apps, appsLoading] = useReapitGet<AppSummaryModelPagedResult>({
+    reapitConnectBrowserSession,
+    action: getActions(window.reapit.config.appEnv)[GetActionNames.getApps],
+    queryParams: { showHiddenApps: 'true', developerId, pageSize: 100, pageNumber: 1 },
+    fetchWhenTrue: [developerId],
+  })
+
+  console.log('Developer Apps: ', apps)
```
[the remark this answers is not in the document]
Comment (willmcvay) [reply packages/app-market/src/components/apps-browse/developer-apps-collection.tsx:20]: As above!

File: packages/app-market/src/components/apps-browse/featured-apps-collection.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  const queryParams = filters ? objectToQuery<AppsBrowseConfigItemFilters>(filters) : {}
+
+  const [apps, appsLoading] = useReapitGet<AppSummaryModelPagedResult>({
+    reapitConnectBrowserSession,
+    action: getActions(window.reapit.config.appEnv)[GetActionNames.getApps],
+    queryParams,
+    fetchWhenTrue: [filters],
+  })
+
+  console.log('Featured Apps: ', apps, content)
```
Comment (bashleigh) [inline packages/app-market/src/components/apps-browse/featured-apps-collection.tsx:24]: found another one :p
another one

File: packages/app-market/src/components/apps-browse/hero-apps.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  const queryParams = filters ? objectToQuery<AppsBrowseConfigItemFilters>(filters) : {}
+
+  const [apps, appsLoading] = useReapitGet<AppSummaryModelPagedResult>({
+    reapitConnectBrowserSession,
+    action: getActions(window.reapit.config.appEnv)[GetActionNames.getApps],
+    queryParams,
+    fetchWhenTrue: [filters],
+  })
+
+  console.log('Hero Apps: ', apps, content)
```
Comment (bashleigh) [inline packages/app-market/src/components/apps-browse/hero-apps.tsx:24]: console.log continues

File: packages/app-market/src/components/apps-browse/simple-apps-collection.tsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  const queryParams = filters ? filters : {}
+
+  const [apps, appsLoading] = useReapitGet<AppSummaryModelPagedResult>({
+    reapitConnectBrowserSession,
+    action: getActions(window.reapit.config.appEnv)[GetActionNames.getApps],
+    queryParams,
+    fetchWhenTrue: [filters],
+  })
+
+  console.log('Simple Apps: ', apps, content)
```
Comment (bashleigh) [inline packages/app-market/src/components/apps-browse/simple-apps-collection.tsx:24]: loggggg

**Review by bashleigh — APPROVED** [review]

**Pull request merged**